resources/benchmark-runner.mjs

```diff
@@ -448,7 +448,7 @@ export class BenchmarkRunner {
                 await suite.prepare(this._page);
                 resolve();
             };
-            frame.src = `resources/${suite.url}`;
+            frame.src = `${suite.url}`;
```

Comment:
frame source is now relative to the root of the repo, so that we can point to either `/workloads` or `/resources`